src/hooks.server.helpers.ts

```diff
@@ -10,6 +10,10 @@ const fetchAccessPolicies = async (
   studentId?: string,
 ) => {
   const isNollning = await isNollningPeriod();
+  const member = await prisma.member.findUnique({
+    select: { classYear: true },
+    where: { studentId: studentId },
+  });
```

Comment:
Is it possible to pass the `member` as an argument instead? I _think_ we've already fetched the member when we call this function.

Normally I wouldn't care much about matters of performance until we know it's an issue, but since we're trying to decrease the load on the database at the moment it would be nice.